src/subdomains/supporting/notification/factories/mail.factory.ts

```diff
@@ -256,17 +274,32 @@ export class MailFactory {
                 ? {
                     link: element.params.url,
                     text: specialTag.value,
+                    textSuffix: specialTag.textSuffix,
                   }
                 : undefined,
+            mail:
+              specialTag?.tag === 'mail'
+                ? { mailAddress: specialTag.value, textSuffix: specialTag.textSuffix }
+                : undefined,
             style: element.params?.style ?? MailDefaultStyle,
             text: specialTag?.text ?? text,
           },
         ];
     }
   }
 
-  private parseSpecialTag(text: string): { text: string; tag: string; value: string } | undefined {
-    const match = /^(.*)\[(\w*):(\w*)\]$/.exec(text);
-    return match ? { text: match[1], tag: match[2], value: match[3] } : undefined;
+  private combineConnectedLineElements(suffix: TranslationItem[]): TranslationItem[] {
+    for (let index = 0; index < suffix.length - 1; index++) {
+      if (suffix[index]?.params?.connectNextLine) {
+        suffix[index + 1].key = `${suffix[index].key}&&${suffix[index + 1].key}`;
+        suffix[index] = null;
+      }
+    }
+    return suffix;
+  }
```

Comment:
I wouldn't use this combine logic, but translation params. E.g.:
```ts
`{ key: `${MailTranslationKey.RETURN}.introduction`, params: { reason: `${Util.parseMailKey(MailTranslationKey.RETURN, entity.amlReason)}` } }`
```
and in the translation file
```json
"introduction": "Grund, warum wir zurückerstattet haben:<br>{reason}",
```
then translate
```ts
 this.tNew(element.key, lang, params);
```
where we should also translate all params.

Reply:
But then we have again parameters in the translation json file 🤔 
Thats what we tried to undo with the Refactoring, or? 😅 
The nice thing with the Refactoring is that everyone can edit these files without any knowledge about parameters and mail logic and so on